scripts/establish_conda_env.sh

```diff
@@ -159,11 +162,11 @@ function display_usage()
 	echo '    --py3'
 	echo '    When installing, make raven_libraries use Python 3'
 	echo ''
-        echo ''
-        echo '    --py2'
-        echo '    When installing, make raven_libraries use Python 2'
-        echo ''
-        echo ''
+    echo ''
+    echo '    --py2'
+    echo '    When installing, make raven_libraries use Python 2'
+    echo ''
+    echo ''
```

Comment:
Do we still support py2?

Reply:
I added a deprecated warning... I will modify it later when I modify the regression test machines